packages/EasyTest/src/Coverage/Factory/CoverageResolverFactory.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyTest\Coverage\Factory;
+
+use EonX\EasyTest\Coverage\Resolvers\CloverCoverageResolver;
+use EonX\EasyTest\Coverage\Resolvers\TextCoverageResolver;
+use EonX\EasyTest\Interfaces\CoverageResolverFactoryInterface;
+use EonX\EasyTest\Interfaces\CoverageResolverInterface;
+use InvalidArgumentException;
+
+final class CoverageResolverFactory implements CoverageResolverFactoryInterface
+{
+    private const CLOVER_EXTENSION = 'clover';
```

Comment:
In most cases we name constants with the rule "the more generic the word the closer to the beginnig".
It allows us to sort constants "by groups". Pls follow this rule here too and name it `EXTENSION_CLOVER`. The same for another const.

Reply:
As soon as I've changed a factory to a locator it does not make sense